src/discussions/posts/post-actions-bar/PostActionsBar.jsx

```diff
@@ -45,10 +45,15 @@ function PostActionsBar({
         {intl.formatMessage(messages.addAPost)}
       </Button>
       {inContext && (
-      <>
-        <div className="border-right mr-3 ml-4" />
-        <IconButton src={Close} iconAs={Icon} onClick={handleCloseInContext} alt={intl.formatMessage(messages.close)} />
-      </>
+        <>
+          <div className="border-right mr-3 ml-4" />
+          <IconButton
+            src={Close}
+            iconAs={Icon}
+            onClick={handleCloseInContext}
```

Comment:
Is the implementation of `handleCloseInContext` intended to be done in the future?

Reply:
Yes, because this is intended to be embedded in an iframe in a sidebar and for this to close the sidebar we need to implement a postMessage-based API that will request the embedding app to close the sidebar.